kafka-connect-s3/src/main/java/io/confluent/connect/s3/hooks/BlockingKafkaPostCommitHook.java

```diff
@@ -42,10 +43,11 @@
 public class BlockingKafkaPostCommitHook implements PostCommitHook {
 
   private static final Logger log = LoggerFactory.getLogger(BlockingKafkaPostCommitHook.class);
-  private static final Pattern pattern = Pattern.compile("topics/(\\d+)/");
   private static final DateTimeFormatter timeFormatter =
           DateTimeFormatter.ofPattern("yyMMdd'T'HHmm00");
-  private long partitioneDurationMinutes;
+  private Pattern pattern;
```

Comment:
it's probably better to make the format more human readable E.g. `yyyy-MM-dd'T'HH:mm`

Reply:
done